.github/workflows/scripts/check_docstrings.py

```
@@ -0,0 +1,773 @@
+
+            # Get the lines of the Docstring, strip hard returns
+            valid_lines = lines[docstring_start : docstring_end + 1]
+
+            # Convert the docstring lines to a string
+            docstring = "\n".join(valid_lines)
+
+            # Parse the docstring
+            try:
+                parser = parse(docstring)
+
+            except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)